src/libostree/ostree-sysroot-upgrader.h

```diff
@@ -85,7 +85,8 @@ gboolean ostree_sysroot_upgrader_check_timestamps (OstreeRepo     *repo,
 
 typedef enum {
   OSTREE_SYSROOT_UPGRADER_PULL_FLAGS_NONE = 0,
-  OSTREE_SYSROOT_UPGRADER_PULL_FLAGS_ALLOW_OLDER = (1 << 0)
+  OSTREE_SYSROOT_UPGRADER_PULL_FLAGS_ALLOW_OLDER = (1 << 0),
+  OSTREE_SYSROOT_UPGRADER_PULL_FLAGS_SYNTHETIC = (1 << 1) /* Don't actually do a pull, just check timestamps/changed */
```

Comment:
Do we need this flag though? Seems like it would be cleaner to just seed `new_revision` with the refspec on upgrader init (similar to what we do in rpm-ostree) and then just not call `ostree_sysroot_upgrader_pull()` at all, no?